content/How_To/GUI/MRTK.md

```diff
@@ -0,0 +1,137 @@
+
+See it in action here:
+<Playground id="#43YQHC#2" title="Slate Scene w/ Label" description="A showcase of different uses for HolographicSlates."/>
+
+#### Near Menu
+The `NearMenu` is a `HolographicMenu` with an attached `FollowBehavior` that tries to stay tethered a set distance from the user at all times.
+
+It is a menu designed for use in XR experiences, and by default stays about half an arm's length away. It also rotates to try and stay in the user's field of view.
+
+The `NearMenu` also has a pin button, that when pressed, will freeze the menu in space. The menu can also be grabbed around the edges to move it (it comes with an attached `SixDofDragBehavior`), at which point it becomes pinned in order to maintain its new position. 
+
+#### Hand Menu
+The `HandMenu` is a `HolographicMenu` that is anchored to the user's hand using a `HandConstraintBehavior`.
+
+Its default position lies on the outside of the user's left hand, such that when their palm is up, the menu is facing them and easily accessible by their right hand.
+
+The menu can be customized to appear by default on the right hand or either hand, and can also be anchored to a different quadrant of the hand. Only one menu will be visible at a time though, even if both hands are up. More information on this can be found in the `HandConstraintBehavior`'s section [here](divingDeeper/gui/mrtk#hand-constraint-behavior).
+
+#### Slider 3D
+The `Slider3D` is a customizable slider with a 3D presence. The default range of the slider is from `0` to `1`, though that can be customized, along with the step count and precision.
+
+The `Slider3D` can be created with a backplate, which is determined on creation. After creation, the `minimum` and `maximum` values can be adjusted, as well as the `step` size. 
+
+See it in action here:
+<Playground id="#2PKNJB#7" title="Slider Color Changer" description="A basic example showing how 3D sliders can affect their surroundings."/>
+
+## Behaviors and Gizmos
+Some behaviors and Gizmos are also shared with MRTK components. In MRTK terminology, these are usually referred to as Solvers, Bounds Control, or Object Manipulators. Behaviors and Gizmos are attached to `Nodes`, `TransformNodes`, and `Meshes` in order to provide additional functionality.
+
+More information on `Gizmos` can be found [here](divingDeeper/mesh/gizmo), while behaviors are covered in more detail [here](divingDeeper/behaviors/meshBehaviors).
+
+#### SixDof Drag Behavior
+The `SixDofDragBehavior`, also known as MRTK's `Object Manipulator` component. By creating and attaching it to an object in the scene, users are able to grab, move, and rotate the object. Interacting with a ray-cast input allows the object to be moved in 3-dimansions, while interacting with near interaction adds the ability to rotate the object in-place.

```

Comment:
Spelling: 3-dimensions.

Sentence edit suggestion:
Interacting with a ray-cast input allows an object to be moved in 3-dim**e**nsions, while interacting with near interaction allows for rotation of an object around its center.

Reply:
The wording of that suggestion to me implies that near interaction doesn't support movement in 3 dimensions, only rotation, which would be incorrect. Is that restriction present in MRTK-Unity?

Reply:
Ah, I miswrote. I meant to say "adds the ability to rotate around it's center." 

I was mainly confused with the "in-place" wording.. I think it's good as is.